README.md

```diff
@@ -145,7 +170,7 @@ code, include a script or makefile that will do the job.
 
 ## A2.5
 
-«replace this with your answer»
+main.cpp
```

Comment:
# 10

The question asked you to translate the table into code, so I was expecting a data structure containing the transitions. Otherwise it looks fine (although why you used the name `counter` for the variable that holds the current state baffles me somewhat).